### PR TITLE
fix: switch unit on confirmation screen

### LINKED ISSUE #1792: [Bug]: Tapping 'spending balance' on Transfer Funds pages doesn't react as defined in design spec
### Describe the bug

Our design spec shows a transition when tapping 'SPENDING BALANCE' on **all** the Transfer Funds 'PLEASE CONFIRM', 'SPENDING BALANCE' and 'Receiving Capacity' pages. Tapping here does nothing in the app.

### Reproduce

See screengrab

### Screenshots / Recording

https://github.com/synonymdev/bitkit/assets/74595920/3d3a5fc2-88fc-484d-863d-c922172423bf

https://github.com/synonymdev/bitkit/assets/74595920/2bb9e047-cba2-4646-aca2-d3d73d48cbe0

https://github.com/synonymdev/bitkit/assets/74595920/0abd21fc-64c5-4de2-8afb-e22e44d716c0

### Operating system

Android 13 TKQ1.220829.002

### Bitkit version

2c5b07726fd35cb796a9566c47d0f6c36fda4ef4

### Log output

_No response_

### PR DESCRIPTION
**Pull request by limpbrains**
### Description

- add unit switching on QuickConfirm screen. Had to add `onChangeUnitOutside` callback to make it work
- i18n 
- show LSP and onchain fees in current unit

### Linked Issues/Tasks

closes #1792 #1926

### Type of change

Bug fix

### Tests

Detox test

### CODE REVIEW

**Comment by pwltr** [comment]
> remove custom fee button from Custom and Quick flows. Keyboard now triggered by pressing on the Amount, as it is in Figma

You sure this is how it should work? The design still has this button.

**Comment by limpbrains** [comment]
> > remove custom fee button from Custom and Quick flows. Keyboard now triggered by pressing on the Amount, as it is in Figma
> 
> You sure this is how it should work? The design still has this button.

I've updated the PR. Button is not removed

File: src/screens/Lightning/QuickConfirm.tsx [hunk 1]
```diff
@@ -131,12 +132,12 @@ const QuickConfirm = ({
...
 				</View>
 
-				<View style={styles.amountContainer}>
+				<TouchableOpacity onPress={switchUnit} style={styles.amountContainer}>
 					<Caption13Up style={styles.amountCaption} color="purple">
 						{t('spending_label')}
 					</Caption13Up>
 					<Money sats={spendingAmount} size="displayT" symbol={true} />
-				</View>
+				</TouchableOpacity>
```
Comment (pwltr) [inline src/screens/Lightning/QuickConfirm.tsx]: For consistency this should use `<Pressable />` and only wrap `<Money />` component

File: src/screens/Lightning/CustomSetup.tsx [hunk 2]
```diff
@@ -371,7 +371,6 @@ const CustomSetup = ({
 
 	const onCustomAmount = (): void => {
 		setShowNumberPad(true);
-		setTextFieldValue('0');
```
Comment (pwltr) [inline src/screens/Lightning/CustomSetup.tsx]: I believe this was added after a user asked for it so they don't have to delete the current amount themself

**Review by pwltr — CHANGES_REQUESTED** [review]

File: src/screens/Lightning/QuickConfirm.tsx (hunk 1 above)
Comment (limpbrains) [reply src/screens/Lightning/QuickConfirm.tsx]: fixed

File: src/screens/Lightning/CustomSetup.tsx (hunk 2 above)
Comment (limpbrains) [reply src/screens/Lightning/CustomSetup.tsx]: reverted

**Review by pwltr — APPROVED** [review]

**Pull request merged**